.github/workflows/pr_verification.yml

```diff
@@ -92,3 +92,26 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: test
+
+  bugzilla_check:
+    name: bugzilla reference check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.BUGZILLA_API_TOKEN }}"
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Added masking for the secrets that we are using so that we don't potentially leak them in visible logs.

Reply:
This has to be some of the crazier syntax I've ever seen for configuring log output, but it works. Good thinking on preempting this kind of leak. :+1: